### PR TITLE
Fix crash when named using `:via`

### PR DESCRIPTION
**Pull request by feng19**
The PR #13 didn't test scene what named using `:via`, It will crash when in use.

I'm sorry about that, please help to review it again.

### CODE REVIEW

**Comment by josevalim** [comment]
@feng19 I am thinking we will have to change our strategy here altogether. Instead of having the tab name be `{:via, ...}`, we should probably call `GenServer.whereis(name)`, whenever the name is not an atom, and work with PIDs instead. This way we don't need to encode name and so on, we only take care of working with PIDs. WDYT?

**Comment by feng19** [comment]
I haven't really latched on to what you mean could you explain it again?

**Comment by josevalim** [comment]
@feng19 all of the Broadway pipelines are actually processes behind the scenes, so all pipelines can also be identified by a `pid` I think? If so, instead of working with their names, such as `{:via, ..., ...}`, we should call `GenServer.whereis` on the `:via` and get its PID instead. Then we will show its PID on the tab names and work the PID throughout the dashboard (except for atoms, which we can keep as atoms).

**Comment by feng19** [comment]
Because of the restriction of `Broadway.topology(pipeline)`, only the name of pipeline can be passed.
So all the places that use `Broadway.topology/1` cannot be replaced.

Therefore, what can be done now is to change the value of nav & nav tab without changing `Broadway.topology/1`.

**Comment by josevalim** [comment]
I see, thanks. I will drop just two small notes. Please check if they work and then we can ship it. :)

File: lib/broadway_dashboard/metrics.ex [hunk 1]
```diff
@@ -21,16 +21,20 @@ defmodule BroadwayDashboard.Metrics do
     end
   end
 
-  def server_name(pipeline) do
-    :"BroadwayDashboard.Metrics.#{pipeline}"
+  def server_name(pipeline) when is_atom(pipeline) do
+    :"Elixir.BroadwayDashboard.Metrics.#{inspect(pipeline)}"
```
Comment (josevalim) [inline lib/broadway_dashboard/metrics.ex:25]:
```suggestion
    :broadway_dashboard_#{inspect(pipeline)}"
```

File: lib/broadway_dashboard/metrics.ex [hunk 2]
```diff
@@ -21,16 +21,20 @@ defmodule BroadwayDashboard.Metrics do
...
+  end
+
+  def server_name({:via, registry, term}) when is_atom(registry) do
+    :"Elixir.BroadwayDashboard.Metrics.#{inspect(registry)}.#{inspect(term)}"
```
Comment (josevalim) [inline lib/broadway_dashboard/metrics.ex:29]:
```suggestion
    :"broadway_dashboard_#{inspect(registry)}_#{inspect(term)}"
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**